services/git-service-impl/src/test/java/io/fabric8/launcher/service/git/AbstractGitServiceTest.java

```diff
@@ -272,6 +275,11 @@ public void pushFile() throws Exception {
 
         //When: pushing a README.md file
         final Path tempDirectory = tmpFolder.getRoot().toPath();
+        Git repo = Git.init().setDirectory(tempDirectory.toFile()).call();
+        repo.getRepository().getConfig().setString(ConfigConstants.CONFIG_USER_SECTION, null, ConfigConstants.CONFIG_KEY_EMAIL, "test@redhat.com");
```

Comment:
Not being picky here, but extracting the Config to a variable may be a good idea :)